mkdocs.yml

```
@@ -116,66 +114,37 @@ nav:
-                  - Miscellaneous:
-                      - microservices/device/services/device-onvif-camera/supplementary-info/control-plane-events.md
-                      - microservices/device/services/device-onvif-camera/supplementary-info/getting-started-with-docker-security.md
-                      - microservices/device/services/device-onvif-camera/supplementary-info/ws-discovery.md
-                      - microservices/device/services/device-onvif-camera/supplementary-info/onvif-footnotes.md
-          - Device REST: microservices/device/services/device-rest.md
-          - Device RFID LLRP: microservices/device/services/device-rfid-llrp.md
-          - Device SNMP: microservices/device/services/device-snmp.md
-          - Device UART: microservices/device/services/device-uart.md
-          - Device USB Camera:
-              - microservices/device/services/device-usb-camera/General.md
-              - Walkthrough:
-                  - microservices/device/services/device-usb-camera/walkthrough/setup.md
-                  - microservices/device/services/device-usb-camera/walkthrough/deployment.md
-                  - microservices/device/services/device-usb-camera/walkthrough/general-usage.md
-                  - microservices/device/services/device-usb-camera/walkthrough/custom-build.md
-              - Supplementary Information:
-                  - microservices/device/services/device-usb-camera/supplementary-info/USB-protocol.md
-                  - microservices/device/services/device-usb-camera/supplementary-info/discovery.md
-                  - microservices/device/services/device-usb-camera/supplementary-info/advanced-options.md
-          - Device Virtual: microservices/device/services/device-virtual/Ch-VirtualDevice.md
-          - Remote Deployment:
-              - Non-Secure Mode: microservices/device/remote/Ch-RemoteNonSecure.md
-              - Secure Mode: microservices/device/remote/Ch-RemoteSecure.md
+        - Device Service: microservices/device/DeviceService.md
+        - Getting Started: microservices/device/GettingStarted.md
+        - Configuration: microservices/device/Configuration.md
+        - API Reference: microservices/device/ApiReference.md
+        - Device Service SDK:
+            - Purpose: microservices/device/sdk/Purpose.md
+            - Getting Started: 
+                -  C Device SDK:
+                   - GettingStartedSDK-C: microservices/device/sdk/devicesdk-getting-started/GettingStartedSDK-C.md
+                -  Go Device SDK:
+                   - GettingStartedSDK-Go: microservices/device/sdk/devicesdk-getting-started/GettingStartedSDK-Go.md
+            - APIs:
+                - C Device SDK :
+                   - APIs: microservices/device/sdk/api/CDeviceSDK/CDeviceSDKAPI.md
+                   - Dependencies: 
+                      - IOTech IOT Utilities: microservices/device/sdk/api/CDeviceSDK/Dependencies/CUtilities.md
```

Do we need the extra layer??
```suggestion
                   - C Utilities: microservices/device/sdk/api/CDeviceSDK/Dependencies/CUtilities.md
```

---

updated